C#Backend/InpatientTherapySchedulingProgramTests/IntegrationTests/HoursWorkedServiceControllerTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using InpatientTherapySchedulingProgram.Controllers;
+using InpatientTherapySchedulingProgram.Models;
+using InpatientTherapySchedulingProgramTests.Fakes;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using InpatientTherapySchedulingProgram.Services;
+
+namespace InpatientTherapySchedulingProgramTests.IntegrationTests
+{
+    [TestClass]
+    public class HoursWorkedServiceControllerTests
+    {
+        private List<HoursWorked> _testHoursWorked;
+        private CoreDbContext _testContext;
+        private HoursWorkedService _testService;
+        private HoursWorkedController _testController;
```

Comment:
Same as before. Possibly rename to more descriptive variable names that pertain to the class that is being tested.